dolphinscheduler-datasource-plugin/dolphinscheduler-datasource-api/src/main/java/org/apache/dolphinscheduler/plugin/datasource/api/datasource/AbstractDataSourceProcessor.java

```diff
@@ -128,4 +130,9 @@ public boolean checkDataSourceConnectivity(ConnectionParam connectionParam) {
             return false;
         }
     }
+
+    @Override
+    public List<String> splitAndRemoveComment(String sql) {
```

Comment:
Can you add some new UT for it?

Reply:
Done. Add some UT case for Oracle, Dorid, Mysql.